server/pulp/server/webservices/views/content.py

```diff
@@ -167,7 +170,7 @@ def post(self, request):
 
         :raises: OperationPostponed when an async operation is performed
         """
-        all_orphans = request.body_as_json
+        all_orphans = request.body_as_json or {}
```

Comment:
This makes sense, but it still looks like an expression. It's fine to leave as-is; you've shown it produces the correct result. I learned something. I'm more used to this style:  https://docs.python.org/2.6/reference/expressions.html#conditional-expressions